notebooks/tfx_pipelines/walkthrough/solutions/tfx_walkthrough_vertex.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    )
</pre><p>Consider adding .with_id("data_transform") or something like that. Maybe .with_id("preprocessing")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
<p>Done! I am using for each component name the actual name of the component to avoid making naming choices.</p>